src/apps/vpn/dnshelper.cpp

```diff
@@ -19,10 +19,28 @@ namespace {
 Logger logger("DNSHelper");
 }
 
+const QString BLOCK_ADS_DNS = "100.64.0.1";
+const QString BLOCK_TRACKERS_DNS = "100.64.0.2";
+const QString BLOCK_ADS_TRACKERS_DNS = "100.64.0.3";
+const QString BLOCK_MALWARE_DNS = "100.64.0.4";
+const QString BLOCK_MALWARE_ADS_DNS = "100.64.0.5";
+const QString BLOCK_MALWARE_TRACKERS_DNS = "100.64.0.6";
+const QString BLOCK_MALWARE_ADS_TRACKERS_DNS = "100.64.0.7";
+
 // static
 QString DNSHelper::getDNS(const QString& fallback) {
+  return getDNSDetails(fallback).ipAddress;
+}
+
+// static
+QString DNSHelper::getDNSType() {
+  return getDNSDetails("test").dnsType;
```

Comment:
Whatever is the fallback needs to be documented on the metrics.yaml file.

Reply:
> never appears in a metric

Is it guaranteed to never appear, or could it appear if something goes (terribly) wrong? If the latter, it needs to be documented.

Reply:
Guaranteed. No `dnsData` struct ever puts the actual DNS address in `dnsType`, for privacy. And this function only returns the `dnsType`.

Reply:
Hum, in that case let's rename it to "IMPOSSIBLE" instead of "Test" and add a little comment saying why it is impossible. Why does it even have a fallback if are never falling back though? Can we just remove the fallback then?

Reply:
The current version has a comment and renamed it to `pretendDNS`. I like `IMPOSSIBLE` even better, thanks. Updating that now.